### ISSUE TITLE
`docs/readthedocs.org:pymc-marketing` memory issue

### ISSUE THREAD

**Issue by williambdean** [pymc-labs/pymc-marketing#1286]
Current error its not a code issue, its on the `docs/readthedocs.org:pymc-marketing`:
```sh
writing output... [ 75%] api/generated/pymc_marketing.mmm.mmm.BaseMMM.get_errors
Command killed due to timeout or excessive memory consumption
```

_Originally posted by @cetagostini in https://github.com/pymc-labs/pymc-marketing/issues/1166#issuecomment-2547090669_

**Comment by ColtAllen** [comment]
This seems to happen a lot with `conda` envs: https://github.com/readthedocs/readthedocs.org/issues/6627

**Comment by wd60622** [comment]
> This seems to happen a lot with `conda` envs: https://github.com/readthedocs/readthedocs.org/issues/6627

Interesting. Can we make the required environment without conda then? I saw something about static built docs as well

cc @OriolAbril

**Comment by OriolAbril** [comment]
I had never seen that before. Do we run code while building the docs? If not he fact that pip installing probably ends up with blas unaware pymc should be completely irrelevant. Otherwise running pymc code might end up taking too long

**Comment by wd60622** [comment]
Yeah, we run the plots in the docstrings. That happens upon build

**Comment by twiecki** [comment]
Looks related: https://github.com/readthedocs/readthedocs.org/issues/4942
Perhaps we can get additional resources.

**Comment by OriolAbril** [comment]
I think this was closed by https://github.com/pymc-labs/pymc-marketing/pull/1695 and for archival purposes ended up being a timeout issue more than a memory one.
